#This message is used to send command to tetra
#Conveyor BD Parameter write command

int32 num
int32 data
---
bool command_Result

